app/views/layouts/application.html.slim

```diff
@@ -37,7 +37,8 @@ html.govuk-template lang='en'
 
       .noprint = render 'layouts/timeout'
       
```

Comment:
I don't like checking the path to make a decision here on whether to display the back link.  I'd like to know why we are hiding the back link in this case and if we can generalize.

Reply:
The designs didn't include any back link and we talked about wanting to force the user into updating their preferences.  But I'll double check if this is the behaviour we're after